sphinxcontrib/datatemplates/cli.py

```diff
@@ -36,24 +24,45 @@ def main():
         'template',
         help='the path to the template file',
     )
+    parser.add_argument(
```

Comment:
I can't seem to add a comment on line 19, but it occurs to me that if we reverse the order of the positional arguments then source could be optional (for templates that only use values from the options), or have multiple values (allowing us to process each data file separately, or all together as one data set, we would have to decide). It would also make it easier to write a script that iterates over a bunch of separate data files one at a time because the part of the command line that would change each time is the last argument.